src/main/kotlin/org/jetbrains/research/testspark/display/TestCasePanel.kt

```diff
@@ -410,6 +424,13 @@ class TestCasePanel(
                         return
                     }
 
+                    collectorsData.feedbackSentCollector.logEvent(
+                        collectorsData.id!! + "_" + testCase.id,
```

Comment:
The `getTestId()` method already implements this; better to re-use the existing method.